File: Herencia/ejercicio 1/jefe.py
from persona import Persona

class Jefe(Persona):
    def __init__(self, nombre: str, apellido: str, ci: str, sucursal: str, tipo: str):
        super()._init_(nombre, apellido, ci)
        self.sucursal = sucursal
        self.tipo = tipo

    def mostrar_datos(self) -> str:
        return f"Jefe: {self.nombre} {self.apellido}, CI: {self.ci}, Sucursal: {self.sucursal}, Tipo: {self.tipo}"

File: Herencia/ejercicio 1/Jefe.java
package herencia;
public class Jefe extends Persona {
    private String sucursal;
    private String tipo;

    public Jefe(String nombre, String apellido, String ci, String sucursal, String tipo) {
        super(nombre, apellido, ci);
        this.sucursal = sucursal;
        this.tipo = tipo;
    }

    @Override
    public String mostrarDatos() {
        return "Jefe: " + nombre + " " + apellido + ", CI: " + ci +
               ", Sucursal: " + sucursal + ", Tipo: " + tipo;
    }
}